metacat-main/src/main/java/com/netflix/metacat/main/api/v1/PartitionController.java

```diff
@@ -426,29 +423,17 @@ public void deletePartitions(
         @ApiParam(value = "Request containing the filter expression for the partitions")
         @Nullable @RequestBody(required = false) final GetPartitionsRequestDto getPartitionsRequestDto
     ) {
-        String filterExpression = null;
-        List<String> partitionNames = null;
-        boolean includePartitionDetails = false;
-        boolean includeAuditOnly = false;
-        if (getPartitionsRequestDto != null) {
-            filterExpression = getPartitionsRequestDto.getFilter();
-            partitionNames = getPartitionsRequestDto.getPartitionNames();
-            includePartitionDetails = getPartitionsRequestDto.getIncludePartitionDetails();
-            includeAuditOnly = getPartitionsRequestDto.getIncludeAuditOnly();
-        }
+
         return this.getPartitions(
             catalogName,
             databaseName,
             tableName,
-            filterExpression,
-            partitionNames,
             sortBy,
             sortOrder,
             offset,
             limit,
             includeUserMetadata,
-            includePartitionDetails,
-            includeAuditOnly
+            getPartitionsRequestDto
```

Comment:
For deletions, the includeAuditOnly should be true always.

Reply:
I github display has issues. This change is for public List<PartitionDto> getPartitionsForRequest(.. 

You can click expanding icon to look at the context.

Reply:
ok